modules/rotating-certificate-authority.adoc

```diff
@@ -6,42 +6,65 @@
 [id="rotating-certificate-authority_{context}"]
 = Rotating the etcd certificate
 
-Rotate the `etcd` certificate before it expires.
+Manually rotate the `etcd` certificate before it expires.
```

Comment:
I think we need to mention that this is now done fully automatic by the cluster-etcd-operator